Presentation.Web/app/models/ViewModel/it-system-usage/system-usage-view-model.ts

```diff
@@ -151,6 +157,9 @@
             this.expirationDate = itSystemUsage.expirationDate;
             this.isActive = itSystemUsage.isActive;
             this.active = itSystemUsage.active;
+            this.lastChanged = itSystemUsage.LastChanged;
```

Comment:
remember to add the conversion using the helper so it does not look like this:
![image](https://user-images.githubusercontent.com/13116889/172622630-d0a6d921-648b-417b-8ff2-ba58d4a8d6ea.png)